plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -152,6 +152,11 @@ proc string mayaUsdTranslatorExport_AppendDefaultPrim(string $currentOptions, st
     if (`optionMenuGrp -exists exportDefaultPrim` == 0)
         return $currentOptions;
     
+    // If the first item, "None" is chosen, then we do *not* set the default prim.
+    int $index = `optionMenuGrp -q -select exportDefaultPrim`;
+    if ($index == 1)
+        return $currentOptions;    

```

Comment:
Fixed bug where chosing "None" for the default prim actually authored that the default prim was a prim named "None".